### ISSUE TITLE
The published vanilla-framework NPM package should just be the contents of scss/

### ISSUE THREAD

**Issue by nottrobin** [vanilla-framework/vanilla-framework#694]
When installing `vanilla-framework` through NPM, it is _always_ with the intention of importing it into your local SCSS:

``` scss
@import 'vanilla-framework/scss/vanilla';
@include vanilla
```

Given that the whole of framework is a collection of SCSS files, the `scss/vanilla` part of this seems like a shame. It would be much nicer if you could just do:

``` scss
@import 'vanilla-framework';
@include vanilla
```

Would it be possible to restructure the repository to achieve this?

**Comment by yaili** [comment]
We've reviewed this at the working group meeting and it's been approved.

**Comment by yaili** [comment]
@nottrobin is this a card for you?

**Comment by barrymcgee** [comment]
Resolved by #970

**Comment by nottrobin** [comment]
@barrymcgee Could you explain how #970 or #975 resolves this issue? As far as I can see you still need to do `@import 'vanilla-framework/scss/vanilla'`.

I'm re-opening this until I understand how this is fixed. Once I understand it I'm happy to close it again.

**Comment by barrymcgee** [comment]
@nottrobin My bad, sorry.

**Comment by anthonydillon** [comment]
@nottrobin Could we simply commit a symlink? As to stop this being a breaking change?

**Comment by nottrobin** [comment]
@anthonydillon yes I believe so. I've been meaning to play around with that as soon as I get the chance. But it hasn't happened yet.

**Comment by Caleb-Ellis** [comment]
Split out the NPM package issue (#1836) and the importing issue. Renaming this one to reflect the importing issue.

**Comment by deadlight** [comment]
This is still valid, although we now include from:
`@import "vanilla-framework/scss/build";`

**Comment by nottrobin** [comment]
The ideal implementation of this requires https://github.com/sass/node-sass/issues/2339, which should be landing soon but not quite yet.

But for the time being, this interface isn't too bad:

``` scss
@import 'vanilla-framework/vanilla';
@include vanilla
```

**Comment by nottrobin** [comment]
Thinking about this, I think we should actually just wait until https://github.com/sass/node-sass/issues/2339 is implemented, and then we should create `_index.scss` in the root of the project, which will wither do `@import scss/vanilla` or `scss/_vanilla.scss` should simply move to `_index.scss`.

**Comment by anthonydillon** [comment]
Could we not just create a new file in the root of the project called `_vanilla-framework.scss` which contains:
```scss
@import 'vanilla-framework/scss/vanilla';
```

This would short cut the full path to allow users that have a node_module includePath to just run:
```scss
@import vanilla-framework;
@include vanilla;
```

**Comment by nottrobin** [comment]
@anthonydillon no, the `vanilla-framework` in `@import 'vanilla-framework'` refers to the module folder itself. So if you create `_vanilla-framework.scss` the root, then your interface will become:

``` scss
@import 'vanilla-framework/vanilla-framework';
@include vanilla;
```

**Comment by anthonydillon** [comment]
Of course, ok lets hold off until index is supported.

**Comment by bartaz** [comment]
Seems like currently it's blocked on https://github.com/sass/node-sass/issues/2339

Alternatively we could investigate migrating to dart sass implementation, which is now up to date with most recent SASS additions (index files, modules, etc) https://www.npmjs.com/package/sass

**Comment by bartaz** [comment]
We moved to dart sass for 3.0, we can check if we can use index now.
